### DISCUSSION TITLE
View entire audio transcription of uploaded audio files

### DISCUSSION THREAD

**Discussion by mindspawn** [open-webui/open-webui#2940] [Ideas]
Currently dropping an mp3 in the conversation performs a transcription and stores it in a txt file in the uploads folder. Would be nice to enable visibility to this file from the webui. Perhaps by clicking on the mp3 attachment in the conversation?

**Comment by jhon1466** [comment]
What model do you use for transcriptions?

**Reply by mindspawn** [reply]
Using the builtin whisper model.
